internal/mesh/internal/controllers/register.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package controllers
+
+import (
+	"github.com/hashicorp/consul/internal/controller"
+	"github.com/hashicorp/consul/internal/mesh/internal/controllers/sidecarproxy"
+	"github.com/hashicorp/consul/internal/mesh/internal/controllers/sidecarproxy/cache"
+	"github.com/hashicorp/consul/internal/mesh/internal/controllers/sidecarproxy/mapper"
```

Comment:
Since these two inner packages have very generic names, it might be clearer to use a custom import like `sidecarproxycache`.

Alternatively, move the cache + mapper out into `internal/mesh/internal/mappers` as something like `sidecarproxymapper`? That's the approach I was taking with the routes controller.

Reply:
That makes sense. I suppose I was thinking that these are specific to the sidecar-proxy controller rather than shared for the mesh package but given that this package is called `controllers`, it makes sense to move out.

Reply:
Done